bin/markdown_lint.sh

```diff
@@ -0,0 +1,2 @@
+#!/usr/bin/env sh
+npx markdownlint-cli concepts/**/*.md
```

Comment:
Probably we can get a lot closer just with

```suggestion
npx markdownlint-cli concepts/**/*.md exercises/**/*.md
```